berkeley-function-call-leaderboard/bfcl/eval_checker/eval_runner.py

```diff
@@ -106,16 +226,16 @@ def single_executable_file_runner(
             "total_count": len(model_result),
         },
     )
-    output_file_name = f"BFCL_v2_{test_category}_score.json"
+    output_file_name = f"{VERSION_PREFIX}_{test_category}_score.json"
     output_file_dir = os.path.join(OUTPUT_PATH, model_name)
     write_list_of_dicts_to_file(output_file_name, result, output_file_dir)
 
     return accuracy, len(model_result)
 
 
-def single_relevance_file_runner(handler, model_result, prompt, model_name, test_category):
+def relevance_file_runner(handler, model_result, prompt, model_name, test_category):
```

Comment:
same: rename "runner" to what is it actually "running". Can be done later.

Reply:
Will address in future PRs